src/main/java/net/mcreator/element/converter/fv26/CommandArgumentBlockFixer.java

```diff
@@ -31,8 +30,14 @@
 	@Override
 	public GeneratableElement convert(Workspace workspace, GeneratableElement input, JsonElement jsonElementInput) {
 		Command command = (Command) input;
-		String procedure = jsonElementInput.getAsJsonObject().get("definition").getAsJsonObject().get("onCommandExecuted").getAsJsonObject().get("name").getAsString();
-		command.argsxml = "<xml><block type=\"args_start\" deletable=\"false\" x=\"40\" y=\"40\"><next><block type=\"call_procedure\"><field name=\"procedure\">" + procedure + "</field></block></next></block></xml>";
+		String procedure = jsonElementInput.getAsJsonObject().get("definition").getAsJsonObject()
+				.get("onCommandExecuted").getAsJsonObject().get("name").getAsString();

```

Comment:
The `onCommandExecuted` can be null in cases the procedure was not selected (so the default one), I guess this is the reason of conversion problems? Then you should check if the _definition_ JSON object `has("onCommandExecuted")`.

Reply:
This is surely the reason. I tested multiple times the converter, but I never thought to test it without selecting a procedure.